.pre-commit-config.yaml

```diff
@@ -1,87 +1,32 @@
 # See https://pre-commit.com for more information
-# See https://pre-commit.com/hooks.html for more hooks
-#
+# Minimal pre-commit to be compatible with current GitHub Actions style.yml
+
 default_language_version:
   python: python3.7
 
 default_stages: [commit, push]
 
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.4.0
-    hooks:
-      - id: check-added-large-files
-      - id: check-yaml
-      - id: check-json
-      - id: end-of-file-fixer
-      - id: mixed-line-ending
-      - id: trailing-whitespace
-
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v2.11.0
-    hooks:
-      - id: pyupgrade
-        args: [--py37-plus, --keep-runtime-typing]
-
-  # automatically removes unused imports
-  - repo: https://github.com/myint/autoflake.git
-    rev: v1.4
-    hooks:
-      - id: autoflake
-        args:
-          [
-            --in-place,
-            --expand-star-imports,
-            --ignore-init-module-imports,
-            --remove-all-unused-imports,
-            --remove-unused-variables,
-            --remove-duplicate-keys,
-          ]
-
   - repo: https://github.com/PyCQA/isort
     rev: 5.8.0
     hooks:
       - id: isort
-        args: ["--profile", "black", "--filter-files"]
-
-  - repo: https://github.com/codespell-project/codespell
-    rev: v2.0.0
-    hooks:
-      - id: codespell
-        types_or: [python, rst, markdown, yaml]
+        args: ["--profile", "black", "--filter-files", "--check"]
 
   - repo: https://github.com/ambv/black
     rev: 20.8b1
     hooks:
       - id: black
+        args: ["--check"]
         language_version: python3.7
 
-  - repo: https://github.com/asottile/blacken-docs
-    rev: v1.10.0
-    hooks:
-      - id: blacken-docs
-
-  - repo: https://github.com/pre-commit/pygrep-hooks
-    rev: v1.8.0
-    hooks:
-      - id: python-use-type-annotations
-
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.9.0
     hooks:
       - id: flake8
-        additional_dependencies: [
-            flake8-comprehensions==3.2.2, # for better list comprehensions
-            flake8-docstrings==1.5.0,
-          ]
+        args: ["--ignore=E203,E266,F401", "--max-line-length=88"]
```

Comment:
Shouldn't we use config from the `.flake8` file?

Reply:
okay, uploaded flake8 file to fix this